### PR TITLE
Mise à jour README

### PR DESCRIPTION
**Pull request by Colack**
Salut, désolé pour mon mauvais français. (J'utilise Traduire + Mauvais français.)   

J'ai mis à jour le README pour afficher une version française du README et une version anglaise du README.   
- Colack

### CODE REVIEW

**Comment by Iltotore** [comment]
Thank you for your contribution :tada:. The english version seems fine to me. Can you just change the names to fit localization names like `README-EN.md`?

Also I think the french readme should stay default (`README.md`) as this project is mainly for french students (it is basically a "french" Pascal with minor changes).

**Comment by Colack** [comment]
Ah ok. I'll open a new pull request. Thank you.

File: README-FR.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # CYLang
 
-Le langage de programmation basé sur le pseudo-code de CYTech
+Le langage de programmation basé sur le pseudo-code de CYTech.   
+
+Read the English version [here.]()
```
Comment (Iltotore) [inline README-FR.md:5]: The link to the english README is missing. Can you correct it ? Once done I'll merge the PR.

**Review by Iltotore — CHANGES_REQUESTED** [review]

**Review by Iltotore — CHANGES_REQUESTED** [review]
Can you rename `README-FR.md` to `README.md` ?

**Comment by Colack** [comment]
oh ok one sec

**Comment by Colack** [comment]
Alright, there you go.

**Comment by Iltotore** [comment]
Thank you. I will add the links tomorrow. It is 23h in my country.

**Pull request merged**